app/views/content_items/service_manual_topic.html.erb

```diff
@@ -22,35 +22,43 @@
 
 <div class="grid-row">
   <div class="column-two-thirds">
-    <% if @content_item.visually_collapsed? %>
-      <div class="js-hidden" data-module="accordion-with-descriptions">
-    <% else %>
-      <div>
-    <% end %>
-      <div class="subsection-wrapper">
-
-        <% @content_item.groups.each_with_index do |link_group, idx| %>
-
-          <div class="subsection">
-            <div class="subsection__header">
-              <h2 class="subsection__title"><%= link_group.name %></h2>
-              <p class="subsection__description"><%= link_group.description %></p>
-            </div>
-            <div class="subsection__content" id="subsection_content_<%= idx %>">
-              <ul class="subsection__list">
-                <% link_group.linked_items.each do |linked_item| %>
-                  <li class="subsection__list-item">
-                    <%= link_to linked_item.label, linked_item.href %>
-                  </li>
-                <% end %>
-              </ul>
+    <% if @content_item.has_sections? %>
+      <div<% if @content_item.visually_collapsed? %> class="js-hidden" data-module="accordion-with-descriptions"<% end %>>
+        <div class="subsection-wrapper">
+          <% @content_item.groups.each_with_index do |link_group, idx| %>
+            <div class="subsection">
+              <div class="subsection__header">
+                <h2 class="subsection__title"><%= link_group.name %></h2>
+                <p class="subsection__description"><%= link_group.description %></p>
+              </div>
+              <div class="subsection__content" id="subsection_content_<%= idx %>">
+                <ul class="subsection__list">
+                  <% link_group.linked_items.each do |linked_item| %>
+                    <li class="subsection__list-item">
+                      <%= link_to linked_item.label, linked_item.href %>
+                    </li>
+                  <% end %>
+                </ul>
+              </div>
             </div>
-          </div>
-
-        <% end %>
-
+          <% end %>
+        </div>
       </div>
-    </div>
+    <% else %>
+      <% @content_item.groups.each do |link_group| %>
+        <div class="subsection">
+          <div class="subsection__content">
+            <ul class="subsection__list">
+              <% link_group.linked_items.each do |linked_item| %>
+                <li class="subsection__list-item">
+                  <%= link_to linked_item.label, linked_item.href %>
+                </li>
+              <% end %>
+            </ul>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
```

Comment:
I wonder if the thing that triggers this "should we display a heading or not" question is on a per-group level rather than at the whole-topic level.

How about..

```
<div class="grid-row">
  <div class="column-two-thirds">
    <div<% if @content_item.visually_collapsed? %> class="js-hidden" data-module="accordion-with-descriptions"<% end %>>
      <div class="subsection-wrapper">
        <% @content_item.groups.each_with_index do |link_group, idx| %>
          <div class="subsection">
            <% if link_group.name.present? %>
              <div class="subsection__header">
                <h2 class="subsection__title"><%= link_group.name %></h2>
                <p class="subsection__description"><%= link_group.description %></p>
              </div>
            <% end %>
            <div class="subsection__content" id="subsection_content_<%= idx %>">
              <ul class="subsection__list">
                <% link_group.linked_items.each do |linked_item| %>
                  <li class="subsection__list-item">
                    <%= link_to linked_item.label, linked_item.href %>
                  </li>
                <% end %>
              </ul>
            </div>
          </div>
        <% end %>
      </div>
    </div>
  </div>
```

And it's up to the publisher to make sure there isn't a weird mix of titles and no titles?